Client.Legacy/FluxClient.cs

```diff
@@ -27,8 +28,16 @@ public FluxClient(FluxConnectionOptions options) : base(new RestClient())
             RestClient.AddDefaultHeader("Accept", "application/json");
             if (!string.IsNullOrEmpty(options.Username))
             {
-                RestClient.AddDefaultQueryParameter("u", options.Username);
-                RestClient.AddDefaultQueryParameter("p", new string(options.Password));   
+                if (options.Authentication.Equals(FluxConnectionOptions.AuthenticationType.BasicAuthentication))
```

Comment:
switch equals to `FluxConnectionOptions.AuthenticationType.BasicAuthentication.Equals(options.Authentication)`